web-console/developer/builds/builds.md

```diff
@@ -1,7 +1,22 @@
 # Builds
 The Builds page is a primary nav item in the OpenShift Developer perspective.
 
-Design documentation for the Builds section can be found [here](https://docs.google.com/document/d/1MWUJ9I_YpI-4e4RVM89g7VNKAj-VTrffDl245IglTfs/edit?usp=sharing)
+## Overview
 
-## Note
-Links to google documents are temporary until the documentation is converted to markdown files and added to this design repository.
\ No newline at end of file
+The Builds page is different from the Administrator perspective.  Clicking on the Builds navigation item will display Build Config resources.
+
+## Build Config Resources
```

Comment:
Total noob question, but why is the nav item called "Builds" and not "Build Configs"?

Reply:
@lizsurette great question!  In the Dev perspective we have tried to simplify things.  Users are concerned about their builds, which are accessed via Build Config objects.

UX & PM decided to have the nav item be called Builds since that's what the users are looking for.   Since we are showing build configs, the page title is Build Configs.  The user can then navigate down to builds.

Hope that helps!

Reply:
Makes total sense to me!!